app/src/main/AndroidManifest.xml

```diff
@@ -28,6 +28,7 @@
         android:theme="@style/KSTheme"
         tools:replace="android:supportsRtl">
         <!-- ACTIVITIES -->
+        <activity android:name="androidx.activity.ComponentActivity" /> <!-- Workaround for https://github.com/robolectric/robolectric/pull/4736 -->
```

Comment:
Encountered this error when trying to run any compose UI tests:
`java.lang.RuntimeException: Unable to resolve activity for Intent { act=android.intent.action.MAIN cat=[an droid.intent.category.LAUNCHER] cmp=com.kickstarter.kickstarter/androidx.activity.ComponentActivity } -- see https://github.com/robolectric/robolectric/pull/4736 for details
`
This workaround to add this ComponentActivity to the manifest ended up being the only thing that allowed me to run any compose UI tests.